### PR TITLE
workbench: renamed "folder" to "sub-directory"

### PR DESCRIPTION
**Pull request by LarsGit223**
This PR changes the naming convention in the workbench plugin from "folder" to "sub-directory".

@b4n, @elextr: please give some feedback as you two participated in the mail thread about the naming convention for directories and folders.

### CODE REVIEW

**Comment by elextr** [comment]
LGBI.  I didn't check the code part in detail, and I personally would only have changed the user facing parts, not the internal names, but you have done it now, so thats fine.

A couple of minor unrelated fixes to the readme noted.

File: workbench/README [hunk 1]
```diff
@@ -85,7 +85,7 @@ These are the available items:
   if you right clicked inside of a project. After selecting it a dialog
   will be opened. Chosse the directory which shall be added. After that
```
Comment (elextr) [inline workbench/README]: Choose

File: workbench/README [hunk 2]
```diff
@@ -101,18 +101,18 @@ These are the available items:
 **Directory settings**
   Select this item to change the directory settings. It is only available
   if you right clicked inside of a project directory. In the directory
-  settings you can set a filter which controls the files and folders
+  settings you can set a filter which controls the files and sub-directories
   that shall be displayed or not.
```
Comment (elextr) [inline workbench/README]: More idiomatic to say "will" in this context.

**Comment by LarsGit223** [comment]
@elextr: fixed README also, thanks for reporting the typos.

**Pull request merged**